### PR TITLE
Remove leading `v`s from version numbers.

### LINKED ISSUE #5: asdf latest bats returns nothing
steps to replicate:

```
asdf add bats
asdf latest bats

asdf list all bats
```

and research if we have to remove prefix `v` from versions

### PR DESCRIPTION
**Pull request by jfly**
This fixes https://github.com/timgluz/asdf-bats/issues/5 (and possibly
https://github.com/timgluz/asdf-bats/issues/6?):

    $ asdf latest bats
    1.6.0

### CODE REVIEW

**Comment by jfly** [comment]
Bump. @timgluz, could you take a look? Thanks!

**Comment by timgluz** [comment]
@jfly Many thanks for contribution; I will give a try and merge it;

File: bin/list-all [hunk 1]
```diff
@@ -11,6 +11,10 @@ function sort_versions() {
     LC_ALL=C sort -i -t. -k 1,1 -k 2,2n -k 3,3n -k 4,4n -k 5,5n | awk '{print $2}'
 }
 
+function remove_leading_vs() {
+  sed 's/^v\(.*\)/\1/'
```
Comment (timgluz) [inline bin/list-all:15]: worked like charm, thanks!

**Review by timgluz — APPROVED** [review]
Well done! It worked well;

**Comment by jfly** [comment]
Thank you!

**Pull request merged**